### DISCUSSION TITLE
How to fix up signature of `load_constant` introduced by `InsertConstIgnore`

### DISCUSSION THREAD

**Discussion by cqc-alec** [Quantinuum/hugr#81] [Specification]
https://github.com/CQCL-DEV/hugr/blob/b6c21d9671eb07d1ef2cde2d577bcebf3ade735c/specification/hugr.md?plain=1#L1199

@acl-cqc 
16 May 2023

Note this leaves us in an invalid state IIUC - load_constant has a signature saying it has one value output of type T

@cqc-alec 
16 May 2023

Oops, that won’t do. Should we add a discard following it, or a new `ignore_const` type?

Another option is to allow any number >=0 of outputs from `load_constant`.

**Comment by cqc-alec** [comment]
May be resolved by #74 .

**Comment by acl-cqc** [comment]
Yes, and/or #80, I think this will be fine.

**Comment by acl-cqc** [comment]
I think following #80 we can close this?
